jtelegrambotapi-core/src/main/java/com/jtelegram/api/commands/Command.java

```diff
@@ -13,14 +13,10 @@
 @ToString
 public class Command {
     private final String baseCommand;
-    private final List<String> args;
+    private final String[] args;
     private final TextMessage baseMessage;
 
     public String getArgsAsText() {
-        StringBuilder sb = new StringBuilder();
-
-        args.forEach(s->sb.append(s).append(" "));
-
-        return sb.toString().trim();
+        return String.join(" ", args);
```

Comment:
We had a few reasons to use Lists over Arrays.

- We can force a list to be immutable and reuse.
- It is more OOP compared to Arrays.
- I can guarantee a List won't have a null element, not the same with an Array.
- I can use a List with various collection apis, and use streams without relying on a separate class.